src/test/java/org/fundacionJala/sfdc/LibraryTest.java

```diff
@@ -1,4 +1,4 @@
-package org.fundacionjala.sfdc;
+package org.fundacionJala.sfdc;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)